ruleset/rules/0802-attack-TA0002.xml

```diff
@@ -66,5 +66,17 @@
         </mitre>
     </rule>
 
+    <!-- {"win":{"eventdata":{"serviceType":"user mode service","accountName":"LocalSystem","imagePath":"%systemroot%\\\\TQkHeboM.exe","startType":"demand start","serviceName":"uxoN"},"system":{"eventID":"7045","eventSourceName":"Service Control Manager","keywords":"0x8080000000000000","providerGuid":"{555908d1-a6d7-4695-8e1e-26931d2012f4}","level":"4","channel":"System","opcode":"0","message":"\"A service was installed in the system.\r\n\r\nService Name:  uxoN\r\nService File Name:  %systemroot%\\TQkHeboM.exe\r\nService Type:  user mode service\r\nService Start Type:  demand start\r\nService Account:  LocalSystem\"","version":"0","systemTime":"2021-06-25T18:33:57.953556700Z","eventRecordID":"171981","threadID":"9104","computer":"bankdc.ExchangeTest.com","task":"0","processID":"544","severityValue":"INFORMATION","providerName":"Service Control Manager"}}} -->
+    <rule id="92031" level="12">
+        <if_sid>61138</if_sid>
+        <field name="win.eventdata.imagePath">^%systemroot%\\\\\w+\.exe$</field>
+        <description>New Windows Service Created to start from windows root path. Suspicious event as the binary may have been dropped using Windows Admin Shares</description>
+        <mitre>
+            <id>T1021.002</id>
```

Comment:
I'm not following why is this kind of rule pages, I mean, T1021 it's a Lateral Movement technique (TA0008) Why it's tagged here? if a rule it's part of many tactics, why not to create a different rule page?

Reply:
Well, it also has Service Execution ta so it's TA0002 too. Perhaps we can find other way to sort the rules out

Reply:
I am using TA002 over TA0008 because that's the Tactic mitre highlighted on this step in evaluation results. ut yeah, we can probably create an issue to sort the rules by a different parameter